Controller/MailgunProjectSettings.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Kanboard\Plugin\Mailgun\Controller;
+use Kanboard\Controller\BaseController;
+
+/**
+ *
+ * @author   Max Eisel
+ */
+class MailgunProjectSettingsController extends BaseController
+{
+    /**
+     * Mailgun Project Action Settings
+     *
+     * @access public
+     * @param array $values
+     * @param array $errors
+     */
+    public function show(array $values = array(), array $errors = array())
+    {
+        $this->response->html($this->helper->layout->project('MailgunProject:settings', array(
+            'owners' => $this->projectUserRoleModel->getAssignableUsersList($project['id'], true),
+	    'values' => array(
+                'MailgunProject_catchall'   => $this->projectMetadataModel->get($project['id'], 'MailgunProject_catchall'),

```

Comment:
Please, don't mix camel case and snake case for the metadata key. Use something like `mailgun_catch_all` instead of `MailgunProject_catchall`. (`Project` is redundant since it's a project metadata).